build_omaha.py

```diff
@@ -61,7 +72,9 @@ def Tagging(args, omaha_dir, debug):
   omaha_out_dir = os.path.join(omaha_dir, 'omaha', 'scons-out', last_win_dir)
   apply_tag_exe = os.path.join(omaha_out_dir, 'obj', 'tools', 'ApplyTag', 'ApplyTag.exe')
 
-  tag = 'appguid=APP_GUID&appname=TAG_APP_NAME&needsadmin=prefers&lang=en&ap=TAG_AP'
+  # right now the needsadmin flag is hardcoded, but should be configurable with an env var or command line flag
+  tag = 'appguid=APP_GUID&appname=TAG_APP_NAME&needsadmin=False&lang=en&ap=TAG_AP'
```

Comment:
Please check that `False` will be only used for migration from muon to brave-core.
Normally, it should be `prefers`.

Reply:
I updated that section to use 'prefers' as the default. We will set the env var TAG_ADMIN in Jenkins for the release build for the migration from browser-laptop only. @bsclifton FYI